sql/tsql/TSqlParser.g4

```diff
@@ -3619,442 +3747,639 @@ sign
     | '-'
     ;
 
+keywords:
```

Comment:
As I understand it's a single keyword, not multiple. `keywords` -> `keyword`.

Also, indentation is not unified for keyword rule. It should be 4 spaces instead of none.